content/de/examples/minikube/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:6.14.2
+FROM node:20.13.1
```

Comment:
is updating the Dockerfile for each localizations relevant in this case (adding linter)?